### PR TITLE
Fix longlat on markers (#13)

### PR DESCRIPTION
**Pull request by komachi**
Use same order as specified in coordorder.

### CODE REVIEW

File: src/MapMarker.jsx [hunk 1]
```diff
@@ -24,7 +25,12 @@ class MapMarker extends Component {
     componentDidUpdate (prevProps) {
         const {lat, lon, children, properties, options} = this.props;
 
-        if (lat !== prevProps.lat || lon !== prevProps.lon) {
+        if (
+            (lat !== prevProps.lat || lon !== prevProps.lon) &&
+            this.context.coordorder === 'longlat'
+        ) {
+            this._controller.setPosition([lon, lat]);
+        } else if (lat !== prevProps.lat || lon !== prevProps.lon) {
```
Comment (effrenus) [inline src/MapMarker.jsx]:
```
if (lat !== prevProps.lat || lon !== prevProps.lon) {
    this._controller.setPosition((this.context.coordorder === 'longlat') ? [lon, lat] : [lat, lon]);
}
```
is better, I think.

File: src/MapMarker.jsx [hunk 2]
```diff
@@ -49,7 +55,11 @@ class MapMarker extends Component {
     componentDidMount () {
         const {lat, lon, properties, options} = this.props;
 
-        this._controller = new MarkerController([lat, lon], properties, options);
+        if (this.context.coordorder === 'longlat') {
+            this._controller = new MarkerController([lon, lat], properties, options);
+        } else {
+            this._controller = new MarkerController([lat, lon], properties, options);
+        }
```
Comment (effrenus) [inline src/MapMarker.jsx]: Same here

File: src/MapContainer.jsx [hunk 3]
```diff
@@ -46,7 +47,8 @@ class YandexMap extends Component {
 
     getChildContext () {
         return {
-            mapController: this._controller
+            mapController: this._controller,
+            coordorder: this.props.coordorder
```
Comment (effrenus) [inline src/MapContainer.jsx]: Must check `coordorder` or set default value for this prop.

**Comment by komachi** [comment]
Fixed.

**Pull request merged**